samples/subsys/ipc/openamp/prj.conf

```diff
@@ -7,3 +7,4 @@ CONFIG_TIMESLICE_SIZE=1
 CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 CONFIG_OPENAMP=y
+# CONFIG_OPENAMP_SLAVE is not set
```

Comment:
I have a doubt... is this sentence disable the config CONFIG_OPENAMP_SLAVE? because it is set to y by default...
perhaps CONFIG_OPENAMP_SLAVE=n should be better or to set to n  by default

Reply:
done.